Online/RainMeadowModManager.cs

```diff
@@ -8,54 +8,49 @@ namespace RainMeadow
     public class RainMeadowModManager
     {
         public static string[] GetActiveMods()
-        { 
+        {
             return ModManager.ActiveMods
                 .Where(mod => Directory.Exists(Path.Combine(mod.path, "modify", "world"))
-                    || highImpactMods.Contains(mod.id)
-                    || cheatMods.Contains(mod.id))
+                    || blacklistMods.Contains(mod.id))
```

Comment:
this won't work

you have to sync the banlist and the require-list separately

Reply:
handle overlaps by removing from banlist?

Reply:
Tested by having host type in "rwremix" into blacklist. Client had remix enabled and was prevented from joining. Not sure what I'm missing from what you're saying.

Reply:
the banlist is for mods that clients should disable
the requirelist is for mods that clients should enable

these two should not be lumped together

Reply:
yes, and this will require redoing a lot of logic

Reply:
> require (beyond "modify", "region")

world and enum-changing ones, order matters